whereami/k8s-grpc/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
               - all
           privileged: false
           readOnlyRootFilesystem: true
-        image: us-docker.pkg.dev/google-samples/containers/gke/whereami:v1.2.14
+        image: us-docker.pkg.dev/google-samples/containers/gke/whereami:v1.2.15
         ports:
           - name: grpc
             containerPort: 9090
```

Comment:
Right below, do you need to change `8080` by `8000` according to your changes in this PR?

Reply:
fixing